### ISSUE TITLE
accessibility issue, any character creation option that isn’t the default custom creation selection will cause screen readers to bug out and not read anything.

### ISSUE THREAD

**Issue by DoctorBoomstick** [CleverRaven/Cataclysm-DDA#65580]
### Describe the bug

Recently, it seems like any character creation option, barring the default custom creation style and play now, will cause screen readers to glitch and fail to read any text, even if the menu is exited. When I, for example, go to create a random character, the screen reader will comfortably read the final panel I’m dropped upon. However, should any key be pressed, the screen reader will cease reading any and all text from the game, requiring a reboot of the game in order to resume regular operations. This persists even if the screen reader itself is turned off and rebooted or the character creator is exited by finishing the generation process.
This makes the random character creator functionally unusable, as one cannot even tab back to review the selected options, pick a starting location, or, indeed, do anything in any way that uses a key press, including pressing tab to enter a game session. Indeed, the only way to both save a character generated in this manner and reacquire spoken feedback is to use hard muscle memory to quick save, save and quit, close the program, relaunch the game, and load into the saved character. Pre-saved character templates run into the same exact issues.

### Attach save file

N/A

### Steps to reproduce

1.	Hop onto an installment of, in my case, Ubuntu 20.04.5.
2.	Enable the natively installed Orca screen reader.
3.	Boot up a pure text copy of the game, running in terminal.
4.	Pick either the random character or the pre-saved character options when going to start a new game.
5. Wait until the data loads and you’re plopped upon the final panel of character creation. Press literally any button and try to navigate around, making note of the fact that the screen reader fails to read any and all text.

### Expected behavior

I expect that I’d be able to pick these options, press my keyboard’s not-so-shiny buttons, do game things, and have my screen reader read any information whatsoever.

### Versions and configuration

• OS: Ubuntu 20.04.5
• Game Version: A3dd8c6
• Graphics version: Terminal
• Ingame language: English

**Comment by Qrox** [comment]
When tested with MSYS2 emulating ncurses, after pressing any key in the random character creation menu, all text except the window border disappeared. This seems to be not only an accessibility issue but probably also a UI issue.
